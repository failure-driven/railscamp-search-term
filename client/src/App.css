.App {
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link, a {
  color: #61dafb;
}

.werewolf-guess {
  width: 310px;
  height: 100px;
  font-size: 20px;
  font-weight: bolder;
  margin-top: 20px;
  border-radius: 4px;
  border: 2px solid #999;
  padding: 5px;
}


.werewolf-guess:focus {
  border: 2px solid aqua;
  outline: none;
}

.werewolf-guess::placeholder{
  color: #DADADA;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
