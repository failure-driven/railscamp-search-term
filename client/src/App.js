import React, { Component } from 'react';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h2>Welcome to the Game</h2>
          <button>Start Game</button>
        </header>
      </div>
    );
  }
}

export default App;
