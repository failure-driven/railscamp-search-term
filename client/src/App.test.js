import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders welcome message', () => {
  const wrapper = shallow(<App />);
  const welcome = <h2>Welcome to the Game</h2>;
  expect(wrapper.contains(welcome)).toEqual(true);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});

it('renders a button', () => {
  const wrapper = shallow(<App />);
  const button = <button>Start Game</button>;
  expect(wrapper.contains(button)).toEqual(true);
});