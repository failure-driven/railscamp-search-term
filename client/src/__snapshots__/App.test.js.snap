// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders welcome message 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h2>
      Welcome to the Game
    </h2>
    <button>
      Start Game
    </button>
  </header>
</div>
`;
